### ISSUE TITLE
Ajax Request: ActiveRecord missing options_callback 

### ISSUE THREAD

**Issue by dmolineus** [contao-community-alliance/contao-multicolumnwizard-bundle#66]
I have a multi column wizard which contains several select fields. All options are computed by an `option_callback` which relies on the presence of the active record. However, it's missing when a new row is created by the ajax request.

**Comment by discordier** [comment]
I have never used the `activeRecord` in MCW myself... what is expected to be contained? I suspect the row values of the current row?

I really wonder if this feature was wildly used and therefore wonder how we shall bring it back (IMO your workaround looks much cleaner).

**Comment by dmolineus** [comment]
> I have never used the activeRecord in MCW myself... what is expected to be contained? I suspect the row values of the current row?

I actually assume the same behaviour af of Contao `options_callback`. In Contao the data container contains the current record, it would be great to have access to the current record (outside of the multi column wizard).

Having access to the current record id would be enough, the `$mcw->id` refers to the widget name instead of the id.

> I really wonder if this feature was wildly used and therefore wonder how we shall bring it back (IMO your workaround looks much cleaner).

It only works in the edit action and not in the edit all action (in the referenced case edit all is disabled so it work here) but I need a solution to modifiy the options callback depending on the current record to which the mcw belongs to.

**Comment by zonky2** [comment]
@dmolineus is this the same as this https://github.com/menatwork/MultiColumnWizard/issues/23 ?

**Comment by dmolineus** [comment]
> @dmolineus is this the same as this menatwork/MultiColumnWizard#23 ?

No, my issue is about a BC break between the mcw and the mcw bundle.

**Comment by zonky2** [comment]
... @dmolineus is trying to provide a PR in "next" time
